java/compaction/compaction-job-creation/src/test/java/sleeper/compaction/strategy/impl/SizeRatioCompactionStrategyTest.java

```diff
@@ -45,11 +46,16 @@
 
 public class SizeRatioCompactionStrategyTest {
 
+    private static final Schema DEFAULT_SCHEMA = schemaWithKey("key");
     private final InstanceProperties instanceProperties = createTestInstanceProperties();
     private final TableProperties tableProperties = createTestTableProperties(instanceProperties, schemaWithKey("key"));
+    private final PartitionsBuilder partitions = new PartitionsBuilder(DEFAULT_SCHEMA)
+            .singlePartition("root");
+    private final FileInfoFactory factory = FileInfoFactory.from(partitions.buildTree());
```

Comment:
It might be clearer if we name this fileInfoFactory, since the type isn't declared in the tests where we see usages of this.